core-web/libs/contenttype-fields/src/lib/fields/binary-field/components/dot-binary-field-url-mode/dot-binary-field-url-mode.component.scss

```diff
@@ -0,0 +1,43 @@
+@use "variables" as *;
+
+.url-mode__form {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-3;
+    justify-content: center;
+    align-items: flex-start;
+}
+
+.url-mode__input-container {
+    width: 100%;
+    display: flex;
+    gap: $spacing-1;
+    flex-direction: column;
+
+    input {
+        width: 100%;
+        min-width: 32rem;
+    }
+}
+
+.url-mode__actions {
+    width: 100%;
+    display: flex;
+    gap: $spacing-1;
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.url-mode__actions--main {
+    width: 6.85rem;
+
+    &::ng-deep {
+        p-button .p-button {
+            width: 100%;
+        }
+
+        .p-button-icon-only {
+            border-radius: $border-radius-md;
+        }
```

Comment:
No, it's not rounded by default. In the `_button.scss` file we skip those styles for that kind of button.

https://github.com/dotCMS/core/blob/7a46d590136215eec95986b5b1515f1a87ba32e4/core-web/libs/dotcms-scss/angular/dotcms-theme/components/buttons/_button.scss#L23-L40

Reply:
Should I make it global? @fmontes

Reply:
Icon only buttons goes with the class `p-button-rounded` to make it a circle, so it does not have any border radius

Reply:
What @zJaaal said :D

Reply:
@zJaaal That's good to know. Is that something we implement for our theme? Because in PrimeNg, the icon buttons do not need that class https://primeng.org/button#icons

Reply:
Hey @zJaaal and @fmontes, just wanted to give you a heads up that the `.p-button-rounded` won't work for this case since it has the `border-radius: 50%;` property. I'm actually curious as to why we removed the rounded styles of the icon buttons when PrimeNg sets it by default.


https://github.com/dotCMS/core/blob/457dccf51263ebc7027c3ccfe7c1024c35004cdb/core-web/libs/dotcms-scss/angular/dotcms-theme/components/buttons/_button.scss#L121-L123